### PR TITLE
Replacing akka-stream-contrib links with links to Alpakka

### PR DESCRIPTION
**Pull request by johanandren**
Additionally, replacing links to code with just file and line number with
links containing commit so that they will stay valid if the sources change over time.

### CODE REVIEW

File: _posts/2016-08-29-connecting-existing-apis.md [hunk 1]
```diff
@@ -169,15 +169,15 @@ public void onTimer(Object timerKey) {
...
-FileTailSource.create(logfile, 4096, 0, FiniteDuration.create(250, TimeUnit.MILLISECONDS))
-  // stream individual lines as elements
-  .via(Framing.delimiter(ByteString.fromString("\n"), 4096))
+final Path logfile = FileSystems.getDefault().getPath("/var/log/system.log");
+final FiniteDuration pollingInterval = FiniteDuration.create(250, TimeUnit.MILLISECONDS);
+final int maxLineLength = 4096;
+FileTailSource.createLines(fs.getPath(path), maxLineLength, pollingInterval)
   .to(AmqpSink.simple(settings))
   .run(materializer);
 ```
```
Comment (johanandren) [inline _posts/2016-08-29-connecting-existing-apis.md:183]: I also updated the gist contents that this is linked to.

**Comment by ktoso** [comment]
LGTM 👍

**Comment by drewhk** [comment]
Can you do this for the XML stuff, too?

**Comment by johanandren** [comment]
We haven't moved that to Alpakka yet, but you mean doing that as well?

**Comment by drewhk** [comment]
Ah, I thought they were moved. No, just leave them as they are now. Contrib should point to Alpakka with a link probably.

**Comment by patriknw** [comment]
@johanandren can this be merged, or are we waiting for something more?

**Comment by johanandren** [comment]
Was just waiting for a second LGTM

**Comment by patriknw** [comment]
thanks

**Pull request merged**